cmake/MUQConfig.cmake.in

```diff
@@ -6,16 +6,39 @@
 #  MUQ_INCLUDE_DIRS   - Include directories that MUQ uses
 #  MUQ_COMPILE_DEFINITIONS -Definitions used during compilation (i.e. variables added with add_definitions)
 
+get_filename_component(MUQ_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+
+include(${MUQ_CMAKE_DIR}/muqTargets.cmake)
```

Comment:
what is muqTargets for?

Reply:
it's generated by cmake, and contains information about dependencies and libraries so that when using MUQ as a library you don't need to manually specify tpl locations for example